### ISSUE TITLE
Two-phase properties of LiBr/water; how to express enthalpies of different solutions with respect to a unique reference state

### ISSUE THREAD

**Issue by ingenerg** [CoolProp/CoolProp#1690]
Hi everybody,
I am using CoolProp to study performance of a single effect lithium bromide/water absorption chiller. 
I have 2 issues and I wish somebody is so kind to help me
1. It seems that LiBr/water solution belongs to the section of CoolProp"incompressible" and the wrapper can't provide properties of two-phase and vapor states. Is there any new project currently about implementing properties in the two phase region?
2. I am trying to write enthalpy balances for a component and I need to set the same reference state for all the streams. If it was a heat exchanger, I wouldn't have this need (the only important quantities would be the differences of enthalpy between inlet and outlet streams). The problem is that such component is a generator receiveing external heat and characterized by the following streams:
- outgoing dry saturated steam
- outgoing solution having a LiBr mass fraction x_1
- incoming solution having a LiBr mass fraction x_2

I noticed that I can choose among many option to set references states, but any of these choices leads to set actually different reference state for each stream. In fact each of these options involves setting reference state at saturated liquid, so that outgoing solution will not have the same reference state as incoming solution. But, in order to perform enthalpy balance properly, I need all the mixing species to be referrred to the same state. Can anyone give me kindly a suggestion about how to solve this?

Thanks a lot

**Comment by wandadars** [comment]
@ingenerg  This might be a question that is better suited for the [user group](https://groups.google.com/forum/#!forum/coolprop-users)

**Comment by ibell** [comment]
@jowr - this is another enthalpy of mixing problem, right?

**Comment by jowr** [comment]
Yes, we do not have the enthalpy of mixing. You could implement that as an extra function though.

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
